Sources/Widgets/Widgets3D/SplineWidget/behavior.js

```diff
@@ -5,31 +5,10 @@ export default function widgetBehavior(publicAPI, model) {
   model.classHierarchy.push('vtkSplineWidgetProp');
 
   // --------------------------------------------------------------------------
-  // Display 2D
-  // --------------------------------------------------------------------------
-
-  publicAPI.setDisplayCallback = (callback) =>
-    model.representations[0].setDisplayCallback(callback);
-
-  // --------------------------------------------------------------------------
-  // Public methods
-  // --------------------------------------------------------------------------
-
-  publicAPI.getPoints = () =>
-    model.representations[1]
-      .getOutputData()
-      .getPoints()
-      .getData();
-
-  // --------------------------------------------------------------------------
-
-  publicAPI.sethandleSizeInPixels = (handleSizeInPixels) => {
-    model.handleSizeInPixels = handleSizeInPixels;
-  };
-
+  // Private methods
   // --------------------------------------------------------------------------
 
-  publicAPI.updateHandlesSize = () => {
+  const updateHandlesSize = () => {
```

Comment:
does not seem to work (anymore?)

Reply:
Can you expand on your issue? It seems to work here:

![ezgif-4-f5d7ac9c53f0](https://user-images.githubusercontent.com/4910855/81192111-d4193480-8f87-11ea-8ca8-425d9e5f8774.gif)

It does not update with camera changes, though that's not a regression since it didn't beforehand either. I know @floryst is looking into bringing some scalable handle representation from Glance back to vtk.js, so we could wait on that and leverage them.

Reply:
My bad, I thought that `updateHandesSize` was for the latter case. Nevermind then.